fireblocks_sdk/sdk.py

```diff
@@ -47,6 +47,70 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
         self.token_provider = SdkTokenProvider(private_key, api_key)
         self.timeout = timeout
 
+    def get_nft(self, token_id: str):
+        url = "/v1/nfts/tokens/" + token_id
+
+        return self._get_request(url)
+
+    def get_nfts(self, token_ids: str):
+        """
+        Example list: "1,2,3,4"
+        """
+        url = f"/v1/nfts/tokens"
+
+        params = {"ids": token_ids}
+
+        if params:
+            url = url + "?" + urllib.parse.urlencode(params)
+
+        return self._get_request(url)
+
+    def refresh_nft_metadata(self, token_id: str):
+        """
+
+        :param token_id:
+        :return:
+        """
+        url = "/v1/nfts/tokens/" + token_id
+        return self._put_request(path=url)
+
+    def refresh_nft_ownership_by_vault(self, blockchain_descriptor: str, vault_id: str):
+        """
+
+        :param blockchain_descriptor:
+        :param vault_id:
+        :return:
+        """
+        url = "/v1/nfts/ownership/tokens"
+        body = {
+            "blockchainDescriptor": blockchain_descriptor,
+            "vaultAccountId": vault_id
+        }
+
+        return self._put_request(path=url, body=body)
+
+    def get_owned_nfts(self, blockchain_descriptor: str, vault_account_id: str, token_ids: str = ""):
+        """
+
+        """
+        url = f"/v1/nfts/ownership/tokens"
+
+        params = {}
+
+        if blockchain_descriptor:
+            params['blockchainDescriptor'] = blockchain_descriptor
+
+        if vault_account_id:
+            params['vaultAccountId'] = vault_account_id
+
+        if token_ids:
+            params['ids'] = token_ids
+
+        if params:
+            url = url + "?" + urllib.parse.urlencode(params)
```

Comment:
WDYT for having a function that do this?

```
if params:
            url = url + "?" + urllib.parse.urlencode(params)
```

So you can reuse it

Reply:
Moved to GET func